subprojects/docs/src/docs/userguide/what_is_gradle.adoc

```diff
@@ -16,34 +16,71 @@
 [[what_is_gradle]]
 = What is Gradle?
 
-Gradle is an open-source https://en.wikipedia.org/wiki/Build_automation[build automation] tool flexible enough to build almost any type of software.
-Gradle makes few assumptions about what you're trying to build or how to build it.
-This makes Gradle particularly flexible.
+Gradle is a https://en.wikipedia.org/wiki/Build_automation[build automation] tool.
 
-== Design
+== Gradle Build Tool
 
-Gradle bases its design on the following fundamentals:
+Gradle Build Tool is the fast and dependable open source build system that automates building software of any type, size or complexity.
+
+== Gradle Enterprise
+
+https://gradle.com/[Gradle Enterprise] is a powerful subscription based tool for managing and monitoring complex software builds. It provides teams with centralized visibility into their build process to quickly identify and address issues as they arise.
```

Comment:
We tend to use one sentence per line in `.adoc` files to make it easier to read diffs when making changes.

Reply:
Noted. Thank you for this tip!